services/QuillLMS/spec/workers/populate_all_activity_healths_worker_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe PopulateAllActivityHealthsWorker do
+  let(:subject) { described_class.new }
+
+  describe '#perform' do
+
+    it 'should track began teacher premium' do
```

Comment:
Nit: After looking at the implementation, I expect this spec to be named something like "it should call PopulateActivityWorker for each activity"

Reply:
Totally copy pasted this from another spec and forgot to change it, sorry!